SimMuon/GEMDigitizer/src/GEMSignalModel.cc

```diff
@@ -21,18 +21,9 @@ GEMSignalModel::GEMSignalModel(const edm::ParameterSet& config)
       timeJitter_(config.getParameter<double>("timeJitter")),
       signalPropagationSpeed_(config.getParameter<double>("signalPropagationSpeed")),
       bx0filter_(config.getParameter<bool>("bx0filter")),
-      digitizeOnlyMuons_(config.getParameter<bool>("digitizeOnlyMuons")),
       resolutionX_(config.getParameter<double>("resolutionX")),
-      muonPdgId(13),
       cspeed(geant_units::operators::convertMmToCm(CLHEP::c_light)),
-      momConvFact(1000.),
-      elecMomCut1(1.96e-03),
-      elecMomCut2(10.e-03),
-      elecEffLowCoeff(1.7e-05),
-      elecEffLowParam0(2.1),
-      elecEffMidCoeff(1.34),
-      elecEffMidParam0(-5.75e-01),
-      elecEffMidParam1(7.96e-01) {}
+      energyMinCut(28.1e-09) {}
```

Comment:
This was before I joined gem, but looking at Seugli's slides, it looks like it was parameterised from geant.

Reply:
Ok, there a reference to page 7 of https://twiki.cern.ch/twiki/pub/CMS/RadiationBackgroundForGE11JINST/JINST_20_035_V16.pdf

Would you mind adding one line of comment that summarizes it then, instead of letting it there as a magic number?

Reply:
yes, we can add in a comment about where the energyMinCut value comes from